### ISSUE TITLE
client container doesn't start

### ISSUE THREAD

**Issue by dleute** [api-platform/api-platform#1691]
Exits with this status:

```
docker logs abc6fed3af52
yarn run v1.22.5
error Command "dev:ssl" not found.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
yarn run v1.22.5
error Command "dev:ssl" not found.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

I can't find any steps in the getting started document that I didn't do.

Other containers are up and appear to be working fine.

I assume I would probably replace this container with a generated progressive web app anyway? But it would be nice if the docs lead to a functioning launch.

**Comment by dleute** [comment]
Actually, the package instructions all exec into this container to generate the client code.

**Comment by dleute** [comment]
I started over, cleared all containers and followed the instructions again and this time the client container came up. Intermittent failed build?

**Comment by mxr576** [comment]
This worked for me: https://github.com/api-platform/api-platform/issues/1686#issuecomment-728373815

**Comment by bertrandjamin** [comment]
I had a similar error few months ago : https://github.com/api-platform/api-platform/issues/1503#issuecomment-616666174

Forcing the build seems to work fine in this case

**Comment by dunglas** [comment]
This should be fixed in 2.6 alpha. We revamped totally how it works.
